<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="2" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-17T17:09:24 IST" name="Suite" finished-at="2022-12-17T17:10:45 IST" duration-ms="81294">
    <groups>
    </groups>
    <test started-at="2022-12-17T17:09:24 IST" name="Test" finished-at="2022-12-17T17:10:45 IST" duration-ms="81294">
      <class name="com.edusoln.runner.TestNGRunner">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:09:24 IST" name="beforeTest" finished-at="2022-12-17T17:09:24 IST" duration-ms="262" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="browserSetUp()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:09:25 IST" name="browserSetUp" finished-at="2022-12-17T17:09:38 IST" duration-ms="13425" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetUp -->
        <test-method signature="testcase(java.util.Map)[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:09:38 IST" name="testcase" data-provider="dp" finished-at="2022-12-17T17:09:38 IST" duration-ms="28" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EmailID=No, User id=Admin, Execute=NO, TestCaseObjective=Verify Login Works fine, AddEmployee=No, FirstName=No, LastName=No, MiddleName=No, Test case no=1, Password=admin123}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case is not run]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case is not run
at com.edusoln.runner.TestNGRunner.testcase(TestNGRunner.java:77)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:09:38 IST" name="tearDown" finished-at="2022-12-17T17:09:39 IST" duration-ms="328" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="browserSetUp()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:09:39 IST" name="browserSetUp" finished-at="2022-12-17T17:09:50 IST" duration-ms="11645" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetUp -->
        <test-method signature="testcase(java.util.Map)[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:09:50 IST" name="testcase" data-provider="dp" finished-at="2022-12-17T17:10:11 IST" duration-ms="21117" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EmailID=100, User id=Admin, Execute=Yes, TestCaseObjective=Verify Add employee Works fine, AddEmployee=Yes, FirstName=Sachin, LastName=Tendulkar, MiddleName=Ramesh, Test case no=2, Password=admin123}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [true] but found [false],
expected [Persona Details] but found [Personal Details]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [Persona Details] but found [Personal Details]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.edusoln.runner.TestNGRunner.testcase(TestNGRunner.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:10:12 IST" name="tearDown" finished-at="2022-12-17T17:10:12 IST" duration-ms="342" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="browserSetUp()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:10:12 IST" name="browserSetUp" finished-at="2022-12-17T17:10:23 IST" duration-ms="11313" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetUp -->
        <test-method signature="testcase(java.util.Map)[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:10:23 IST" name="testcase" data-provider="dp" finished-at="2022-12-17T17:10:44 IST" duration-ms="20848" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EmailID=77, User id=Admin, Execute=Yes, TestCaseObjective=Verify Add employee Works fine, AddEmployee=Yes, FirstName=Sourav, LastName=Ganguly, MiddleName=test, Test case no=3, Password=admin123}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [true] but found [false],
expected [Persona Details] but found [Personal Details]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [Persona Details] but found [Personal Details]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.edusoln.runner.TestNGRunner.testcase(TestNGRunner.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:10:45 IST" name="tearDown" finished-at="2022-12-17T17:10:45 IST" duration-ms="344" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:10:45 IST" name="afterTest" finished-at="2022-12-17T17:10:45 IST" duration-ms="380" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="Suite()[pri:0, instance:com.edusoln.runner.TestNGRunner@2f08c4b]" started-at="2022-12-17T17:10:45 IST" name="Suite" finished-at="2022-12-17T17:10:45 IST" duration-ms="18" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Suite -->
      </class> <!-- com.edusoln.runner.TestNGRunner -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
